generators/module/index.js

```diff
@@ -49,16 +49,23 @@ module.exports = class extends Generator {
       type: 'input',
       name: 'userName',
       message: 'Your name',
-      default: username.substring(0, username.length - 1)
+      default: username.substring(0, username.length - 1),
+      store: true
```

Comment:
please no. you should store your .yorc in your home directory, not in the project